packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.base.tsx

```diff
@@ -109,7 +110,7 @@ export class CheckboxBase extends React.Component<ICheckboxProps, ICheckboxState
               onFocus={this._onFocus}
               onBlur={this._onBlur}
               aria-disabled={disabled}
-              aria-label={ariaLabel}
+              aria-label={ariaLabel || label}
```

Comment:
Just wondering if this change is really necessary? None of the examples seem to explicitly define the aria-label, it seems to read the label just fine atm (probably based on the `for` association)

Reply:
I would have thought that the for association would be enough, but in my testing, without this change the name of the checkbox doesn't end up getting read unless the ariaLabel is explicitly set. I think now that the label is marked as hidden, it might no longer be getting used properly in this context.

Reply:
That makes sense, thanks for providing this context.